apps/dataset-browser/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Do we need this `.gitignore`? Or can we use the one in the root?

Reply:
Lets use just one